app/models/effective/datatables/employer_datatable.rb

```diff
@@ -111,8 +112,18 @@ def nested_filter_definition
         @next_30_day = TimeKeeper.date_of_record.next_month.beginning_of_month
         @next_60_day = @next_30_day.next_month
         @next_90_day = @next_60_day.next_month
+        upcoming_dates = [
```

Comment:
```
Config::SiteHelper
   def number_of_months_application_can_be_made_in_advance
      Settings.aca.shop_market.initial_application.earliest_start_prior_to_effective_on.months.abs
  end
end

EmployerDatatable
  upcoming_dates = [
     { scope: TimeKeeper.date_of_record.next_month.beginning_of_month, 
        label: TimeKeeper.date_of_record.next_month.beginning_of_month 
     }
 ]
  number_of_months_application_can_be_made_in_advance.times do |n|
     next_month = upcoming_dates.last[:scope].next_month
     upcoming_dates.append({ scope: next_month, label: next_month })
  end
end
```

Reply:
something like that I think

Reply:
I tried as you suggested. Please do review it again. Thank you @brianweiner